src/ctl/src/lib.rs

```diff
@@ -72,6 +72,9 @@ enum Commands {
     /// Dump the await-tree of compute nodes and compactors
     #[clap(visible_alias("trace"))]
     AwaitTree,
+    /// Detect the bottleneck actors from the await-tree of compute nodes
+    #[clap(visible_alias("trace-analyze"))]
+    AwaitTreeAnalyze,
```

Comment:
Prefer to make this a subcommand of `AwaitTree`

Reply:
updated